app/helpers/message_helper.rb

```diff
@@ -3,9 +3,9 @@ def self.export_filename(message)
     "#{message.delivered_at.to_date}-sprava-#{message.id}.zip"
   end
 
-  def self.format_html_visualization(message)
-    return ActionController::Base.helpers.simple_format(message.html_visualization) if message.is_a?(Fs::MessageDraft)
+  def format_html_visualization
+    return ActionController::Base.helpers.simple_format(html_visualization) if is_a?(Fs::MessageDraft)
```

Comment:
Myslim, ze tu uz nepotrebujes to volat takto cez `..Base` ale malo by stacit priamo zavolat `simple_format`

Reply:
Nie, skusala som